build.sbt

```diff
@@ -28,8 +28,10 @@ version.in(ThisBuild) ~= { old =>
 }
 name := {
   println(s"[info] Welcome to scalameta ${version.value}")
-  val javaVersion = sys.props("java.specification.version")
-  if (javaVersion != "1.8") sys.error(s"Obtained Java version $javaVersion. Expected 1.8")
+  if (!sys.props.isDefinedAt("scalameta.allow-any-jdk")) {  // escape hatch for Scala community build
+    val javaVersion = sys.props("java.specification.version")
+    if (javaVersion != "1.8") sys.error(s"Obtained Java version $javaVersion. Expected 1.8")
+  }
```

Comment:
@olafurpg Do you remember why we added this check?

Reply:
I don't remember, it could be this assertion is to guard against errors on JDK 7. I prefer to keep the assertion by default since users on Java 9-10 are likely to encounter errors and I prefer to address the breaking changes once Java 11 is out.